Dockerfile

```diff
@@ -233,7 +233,7 @@ RUN export COMPOSER_HOME=/composer && \
     chown -R www-data:www-data /opt/kimai && \
     sed "s/128M/256M/g" /usr/local/etc/php/php.ini-development > /usr/local/etc/php/php.ini && \
     sed "s/128M/-1/g" /usr/local/etc/php/php.ini-development > /opt/kimai/php-cli.ini && \
-    sed -i "s/env php/env -S php -c /opt/kimai/php-cli.ini/g" /opt/kimai/bin/console
+    sed -i "s/env php/env -S php -c \/opt\/kimai\/php-cli.ini/g" /opt/kimai/bin/console
```

Comment:
What about the `.` and `-`: could they need escaping as well?

Reply:
No, the / was a delimiter of the sed search.  I should have seen it but it caused the sed search to end at the slash before opt

Reply:
Yep, I see it now as well^^ But thats a regex and doesn't have the `.` a special meaning?
Well, if it works I assume all is good.

Reply:
You're right the dot does but only in the search part, in the replacement it's treated as just a character.

Reply:
Oh right, totally missed that ... yes that line is a little bit confusing to read ;-)